src/Receipt/DonorDetailsGroup/Details/Name.php

```diff
@@ -0,0 +1,37 @@
+<?php
+namespace Give\Receipt\DonorDetailsGroup\Details;
+
+use Give\Receipt\Detail;
+use function give_get_payment_meta as getDonationDonorName;
+
+/**
+ * Class Name
+ *
+ * @since 2.7.0
+ * @package Give\Receipt\DonorDetailsGroup\Details
+ */
+class Name extends Detail {
+	/**
+	 * @inheritDoc
+	 */
+	public function getLabel() {
+		return __( 'DONOR NAME', 'give' );
```

Comment:
Because translated strings are user-defined, the translated string should be escaped.